main.tex

```diff
@@ -722,33 +723,31 @@
 If a robot can store, earn, spend and invest - they can do everything you can do
 \end{lstlisting}
 
-What is needed is a simple, yet a powerful state reality tool with the ability to find particular elements. \linkred{https://github.com/cybercongress/go-cyber}{go-cyber} offers minimalistic, but continuously self-improving data source, which provides the necessary tools for programming economically rational robots. According to \linkgreen{https://github.com/first20hours/google-10000-english}{top-10,000 English words} the most popular word in the English language is the defining article 'the', which means a pointer to a particular item. This fact can be explained as the following: particular items are of most importance to us. Therefore, the nature of our current semantic computing is to find unique things. Hence, the understanding of unique things is essential for robots too.
+What is needed is a simple, yet a powerful state reality tool with the ability to find particular elements. \linkred{https://github.com/cybercongress/go-cyber}{go-cyber} offers minimalistic, but continuously self-improving data source, which provides the necessary tools for programming economically rational robots. According to \linkred{https://github.com/first20hours/google-10000-english}{top-10,000 English words} the most popular word in the English language is the defining article 'the', which means a pointer to a particular item. This fact can be explained as the following: particular items are of most importance to us. Therefore, the nature of our current semantic computing is to find unique things. Hence, the understanding of unique things is essential for robots too.
```

Comment:
the ability to find particular elements => particular things

Reply:
changed